packages/panic/package.json

```diff
@@ -0,0 +1,69 @@
+{
+  "name": "@endo/panic",
+  "version": "0.1.0",
+  "private": true,
```

Comment:
I’m fine with merging now since this is marked as not-to-be-published, and I like waiting until we have a concrete use to test in integration before committing to an API/behavior. If you’ve got one in mind now, let’s flip this bit.

Reply:
I do! Repairing the ERTP safety issue reported by Felipe Natal. Draft at https://github.com/Agoric/agoric-sdk/pull/11173 I will fix once agoric-sdk is synced with an endo with this ponyfill.

Reply:
Aha! I missed this. Adding to https://github.com/endojs/endo/issues/2816#issuecomment-2896090497

Reply:
Done. In light of #11173 I changed it to `false`.